keyboards/keebmonkey/kbmg68/readme.md

```diff
@@ -0,0 +1,27 @@
+# KBMG68
+
+![KBMG68](https://www.keebmonkey.com/products/kbmg68)
```

Comment:
Needs to be an actual image link.

Reply:
i don't think you understand LOL...this is barely even a product.  but ok.  i'll give a link to the absolutely sh*tty picture they show.

Reply:
`![]()` is Markdown syntax for an image. The URL you've provided is invalid for that.

Reply:
Ah ok it is I who did not understand.  Well anyway, i have updated the md...this is the best image they have and it sucks bro.  Sorry :-|